src/main/kotlin/com/amazon/opendistroforelasticsearch/indexstatemanagement/models/Transition.kt

```diff
@@ -73,16 +75,21 @@ data class Conditions(
     val docCount: Long?,
     val size: String?,
     val cron: CronSchedule?
-) : ToXContent {
+) : ToXContentObject {
 
     init {
-        var conditionsSet = 0
-        if (indexAge != null) conditionsSet++
-        if (docCount != null) conditionsSet++
-        if (size != null) conditionsSet++
-        if (cron != null) conditionsSet++
+        val conditionsList = listOf(indexAge, docCount, size, cron)
+        require(conditionsList.filterNotNull().size < 2) { "Cannot provide more than one Transition condition" }
+
+        // Validate indexAge condition
+        try {
+            TimeValue.parseTimeValue(indexAge, "")
```

Comment:
Assuming this is:
```
public static TimeValue parseTimeValue(String sValue, String settingName) {
        Objects.requireNonNull(settingName);
        Objects.requireNonNull(sValue);
        return parseTimeValue(sValue, null, settingName);
    }
```
Seems like this will always throw NPE when indexAge is null

Reply:
Fixed.